pipelines/jobs/configurations/jdk11u_evaluation.groovy

```diff
@@ -1,4 +1,10 @@
 targetConfigurations = [
+        'aarch64Linux'  : [
```

Comment:
still not Alpine Linux? perhaps I'm missing something here?

Reply:
see it as my covid side effects.......

we actually do not have jdk11 testing  aarch64alpinelinux.
will not have it in evaluation.

revert back

Reply:
there was some issue for jdk11 aarch64 alpine linux in the past, so it was disabled on jdk11
will first get other jdk version running and put it into evaluation later.